plugin/hosts/README.md

```diff
@@ -11,14 +11,18 @@ available hosts files that block access to advertising servers.
 
 ~~~
 hosts [FILE [ZONES...]] {
+    [INLINE]
     fallthrough
 }
 ~~~
 
 * **FILE** the hosts file to read and parse. If the path is relative the path from the *root*
   directive will be prepended to it. Defaults to /etc/hosts if omitted
 * **ZONES** zones it should be authoritative for. If empty, the zones from the configuration block
-    are used.
+   are used.
+* **INLINE** the hosts file contents inlined in Corefile. If there are any lines before fallthrough
```

Comment:
What about conflicts with /etc/hosts? Will they be overwritten? (would be useful I think)

Reply:
At the moment if there are any inlined content, then `/etc/hosts` will be ignored. I think it also make sense to 
1. Combine them
2. Report conflict.

But I am not sure which way to go. Do you have some thought on it?

Reply:
I would think overriding would make sense, but not sure. @pmoroney any ideas?